jicofo-common/src/main/kotlin/org/jitsi/jicofo/xmpp/muc/ChatRoomMemberImpl.kt

```diff
@@ -175,22 +188,6 @@ class ChatRoomMemberImpl(
         presence.getExtension(StatsId::class.java)?.let {
             statsId = it.statsId
         }
-
```

Comment:
Should we not leave this in if the endpoint doesn't support source-names ?

Reply:
Do endpoints with source name support always include `SourceInfo` (even when they don't have any tracks)?

Reply:
Nope, SourceInfo gets added only if there are local tracks.

Reply:
Sorry, just realized that we send an empty SourceInfo on join if the endpoint doesn't have any local tracks. 
"<SourceInfo>{}</SourceInfo>"